mappings/net/minecraft/client/util/math/AffineTransformation.mapping

```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_4590 net/minecraft/client/util/math/Rotation3
 	FIELD field_20905 rotation1 Lnet/minecraft/class_1158;
```

Comment:
What does the `1` mean here?

And why is there a rotation field at all, can't the rotation be included in the matrix?

Reply:
See #1241 and that forge class

Reply:
The Forge class calls them `leftRot` and `rightRot`. Tbh a lot of these names in this class are a mess and warrant a PR of their own.

> And why is there a rotation field at all, can't the rotation be included in the matrix?

It is. It's a cached version that may be recomputed in the getter.

Reply:
These fields are really only initialized but never used... The only `getRotation2` is retained because when `IDENTITY` is constructed, `getRotation2` is called to initialize the components

Reply:
There were probably such getters in the original source which proguard stripped because they weren't used elsewhere.

Reply:
Can I leave this field rename till later because these fields don't really matter?